Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Winium.StoreApps.Driver.CommandExecutors
+{
+    #region
+
+    using System.Collections.Generic;
+
+    using Winium.StoreApps.Common;
+    using Winium.StoreApps.Driver.Helpers;
+
+    #endregion
+
+    internal class ExecuteScriptExecutor : CommandExecutorBase
+    {
+        #region Methods
+
+        internal object ExecuteStorageScript(string command)
+        {
+            switch (command)
+            {
+                case "ReadLocalTextFile":
+                    var args = ExecutedCommand.Parameters["args"] as JArray;
+                    if (args == null || args.Count == 0)
+                    {
```

Comment:
I would agree, but that's not how it's done in the rest of the Winium project, so I'll stick to consistency.